src/index.ts

```diff
@@ -39,19 +43,47 @@ githubApp.webhooks.on("workflow_job.queued", async (event) => {
                 status: "queued"
             })
         };
-    response = await fetch(database+`/api/config_repos/${event.payload.repository.name}`, requestOptions);
-    
+
+    var response = await fetch(database+`/api/config_repos/${event.payload.repository.name}`, requestOptions);
+
     // call API to compute service
+    // generate token based on org name and repository
+    const secretKey = await fs.readFile(path.join(__dirname, 'src/middleware/custom-token.pem')) 
+    const repoName = event.payload.repository.html_url.split("/").at(-1);
+    const accessToken = jwt.sign(
+            {org: event.payload.sender.login,
+             reponame: repoName},
+             secretKey,
+            {expiresIn: "30m"}
```

Comment:
I think that in some circumstances we'll want to use longer expirations - could you please make this be configurable via an environmental variable?

Reply:
Update to env variable